src/utils/IPFSUtils.ts

```diff
@@ -0,0 +1,84 @@
+import PinataClient from "@pinata/sdk";
+import axios from "axios";
+import { BigNumber } from "ethers";
+
+/**
+ * Build an IPFS client for interacting with the IPFS API
+ * @param APIKey The project ID
+ * @param secretAPIKey The project secret
+ * @returns An IPFS client for interacting with Pinata
+ */
+export function buildIPFSClient(APIKey: string, secretAPIKey: string): PinataClient {
+  return new PinataClient(APIKey, secretAPIKey);
+}
```

Comment:
Curious about this wrapper - it seems unnecessary. Do you anticipate that it could be expanded in future?

Reply:
I was trying to put as much logic in necessary utils & the class itself is for the caching interface.

Reply:
Agreed